evm/src/executor/inspector/fuzzer.rs

```diff
@@ -82,12 +82,13 @@ impl Fuzzer {
             state.insert(utils::u256_to_h256_be(*slot).into());
         }
 
-        for index in 0..interpreter.memory.len() / 32 {
-            let mut slot = [0u8; 32];
-            slot.clone_from_slice(interpreter.memory.get_slice(index * 32, 32));
+        // TODO: disabled for now since it's flooding the dictionary
+        // for index in 0..interpreter.memory.len() / 32 {
+        //     let mut slot = [0u8; 32];
+        //     slot.clone_from_slice(interpreter.memory.get_slice(index * 32, 32));
```

Comment:
Is it actually? The dictionary is a HashSet, so duplicate values will not affect it at all

Reply:
It's adding values which are not properly aligned/parsed. An address `00....1337` is most likely to be inserted as `00...133700000000000000002200000000003300000000000000`